### ISSUE TITLE
Installed (but unpackaged) file(s) found: /usr/bin/pybabel

### ISSUE THREAD

**Issue by patsevanton** [fedora-python/pyp2rpm#220]
```
cat /etc/redhat-release 
CentOS Linux release 7.6.1810 (Core)
pip install --user pyp2rpm
pyp2rpm Babel -t epel7 -b2 -p2 > Babel-2.8.0.spec
```
Babel-2.8.0.spec:
```
# Created by pyp2rpm-3.3.3
%global pypi_name Babel

Name:           python-%{pypi_name}
Version:        2.8.0
Release:        1%{?dist}
Summary:        Internationalization utilities

License:        BSD
URL:            http://babel.pocoo.org/
Source0:        https://files.pythonhosted.org/packages/source/B/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
BuildArch:      noarch

BuildRequires:  python2-devel
BuildRequires:  python2-pytz >= 2015.7
BuildRequires:  python2-setuptools
BuildRequires:  python2-sphinx

%description
A collection of tools for internationalizing Python applications.

%package -n     python2-%{pypi_name}
Summary:        Internationalization utilities

Requires:       python2-pytz >= 2015.7
%description -n python2-%{pypi_name}
A collection of tools for internationalizing Python applications.

%package -n python-%{pypi_name}-doc
Summary:        Babel documentation
%description -n python-%{pypi_name}-doc
Documentation for Babel

%prep
%autosetup -n %{pypi_name}-%{version}
# Remove bundled egg-info
rm -rf %{pypi_name}.egg-info

%build
%{__python2} setup.py build
# generate html docs
PYTHONPATH=${PWD} sphinx-build docs html
# remove the sphinx-build leftovers
rm -rf html/.{doctrees,buildinfo}

%install
%{__python2} setup.py install --skip-build --root %{buildroot}

%check
%{__python2} setup.py test

%files -n python2-%{pypi_name}
%doc README.rst
%{python2_sitelib}/babel
%{python2_sitelib}/%{pypi_name}-%{version}-py%{python2_version}.egg-info

%files -n python-%{pypi_name}-doc
%doc html
%license LICENSE docs/_themes/LICENSE docs/license.rst

%changelog
* Sat Feb 22 2020 Cloud User - 2.8.0-1
- Initial package.
```

https://gist.github.com/patsevanton/e437aa6fa89f9119d33a5e88c4b8dbd3

Is it bug or not?

**Comment by jcrd** [comment]
I am experiencing this same issue in attempting to package [lesspass](https://pypi.org/project/lesspass/).
`%{_bindir}/lesspass` is absent from the `%files` section in the generated spec file.

Strangely, this was not the case ~5 months ago when the [copr build](https://copr.fedorainfracloud.org/coprs/jcrd/lesspass/build/1575887/) succeeded, as can be seen in the [spec file](https://copr-be.cloud.fedoraproject.org/results/jcrd/lesspass/fedora-33-x86_64/01575887-python-lesspass/python-lesspass.spec) (created by `pyp2rpm-3.3.2`).

**Comment by gordonmessmer** [comment]
@jcrd Are you sure that spec was entirely generated?  If I install pyp2rpm 3.3.2 on CentOS 7, with python 3.6, and if I modify it just to rate limit calls to the deprecated XMLRPC API, that version of pyp2rpm will generate a spec file for lesspass 10.0.0 that doesn't include %{_bindir}/lesspass in the files section.

If we can find conditions under which an older version builds a working spec, that'll really help in identifying the cause of the change.

**Comment by gordonmessmer** [comment]
I think I understand... These look like components that are only extracted if the virtualenv and virtualenv-api python modules are installed.  Please ensure those two modules are installed and see if you can reproduce the problem.

**Comment by jcrd** [comment]
The problem persists with these modules installed:
```
> pip list | grep 'pyp2\|virtual'
pyp2rpm                3.3.5
virtualenv             20.2.2
virtualenv-api         2.1.16
virtualenv-clone       0.5.4
```

**Comment by gordonmessmer** [comment]
This should be fixed in #255 .  The only change strictly necessary to correct it should be to remove "self.version" from the call to env.install on line 89 of virtualenv.py:

https://github.com/fedora-python/pyp2rpm/pull/255/files#diff-c9a4007c6badaaac14e6fa79d397d508a966a632c4384e9ef68b17d5fab2f59bL89
